Assets/Scripts/GameSelection.cs

```diff
@@ -81,30 +79,91 @@ public GameSelection()
+                // TODO: FFGlookup to get the first line (languages list)
+                string langsString = StringI18n.FFG_LANGS;
+
+                // List of languages
+                game.config.data.Add("MoMConfig","Langs", langsString);
+                // English is the default current language
+                game.config.data.Add("MoMConfig", "currentLang","1");
+                game.config.Save();
             }
-            else // Import OK, can redo
+
+            string[] langs = game.config.data.Get("MoMConfig", "Langs").Split(',');
+            int currentLang = int.Parse(game.config.data.Get("MoMConfig", "currentLang"));
+
+            game.currentLang = currentLang;
+
+            float verticalStart = 12 - ((langs.Length - 1) * 1.25f);
+
+            languageTextButtons = new TextButton[langs.Length];
+            for (int i = 1; i < langs.Length; i++)
             {
-                tb = new TextButton(new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 23.2f), new Vector2(10, 2f), "Reimport Content", delegate { Import("MoM"); });
-                tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0, 0.03f, 0f);
+                // Need current index in order to delegate not point to loop for variable
+                int currentIndex = i;
+
+                languageTextButtons[currentIndex] = new TextButton(
+                    new Vector2(UIScaler.GetHCenter() + 15, verticalStart + (2.5f * currentIndex)),
+                    new Vector2(8, 2f), 
+                    langs[currentIndex], 
+                    delegate { SelectLang(currentIndex); },
+                    currentIndex == currentLang ? Color.white : Color.gray
+                    );
             }
         }
-        else // Import unavailable
+
+        new TextButton(new Vector2(1, UIScaler.GetBottom(-3)), new Vector2(8, 2), "Exit", delegate { Exit(); }, Color.red);
+    }
+
+    /// <summary>
+    /// Select current language to specified
+    /// </summary>
+    /// <param name="i"></param>
+    private void SelectLang(int i)
+    {
+        Game currentGame = Game.Get();
+        if (currentGame.currentLang < languageTextButtons.Length)
         {
-            db = new DialogBox(new Vector2((UIScaler.GetWidthUnits() - 24) / 2, 23.2f), new Vector2(24, 1f), "Unable to locate Mansions of Madness, install via Steam", Color.red);
-            db.AddBorder();
+            languageTextButtons[currentGame.currentLang].setColor(Color.gray);
         }
-        new TextButton(new Vector2(1, UIScaler.GetBottom(-3)), new Vector2(8, 2), "Exit", delegate { Exit(); }, Color.red);
+        languageTextButtons[i].setColor(Color.white);
+        // English is the default current language
+        currentGame.config.data.Add("MoMConfig", "currentLang", i.ToString());
+        currentGame.config.Save();
+        currentGame.currentLang = i;
+        refreshScreen();
+        Debug.Log("new current language stablished:" + i.ToString());
```

Comment:
Strange unity quirk: For logs like this add System.Environment.NewLine to the end, or it will put stack trace info into what should be just an informative entry.